public/index.html

```diff
@@ -2,7 +2,7 @@
 <html lang="en">
   <head>
     <meta charset="utf-8">
-    <meta http-equiv="Content-Security-Policy" content="connect-src 'self' ws://localhost:3000 https://catalogue.data.govt.nz https://api.addressfinder.io; style-src blob: https://api.addressfinder.io https://unpkg.com https://serviceinnovationlab.github.io 'unsafe-inline'; script-src 'self' https://api.addressfinder.io https://serviceinnovationlab.github.io https://unpkg.com 'unsafe-eval' 'unsafe-inline' 'unsafe-inline'; default-src 'none'; img-src 'self' data: https://a.tile.openstreetmap.org https://b.tile.openstreetmap.org https://c.tile.openstreetmap.org https://serviceinnovationlab.github.io https://unpkg.com; child-src 'none'; base-uri https://serviceinnovationlab.github.io; form-action https://serviceinnovationlab.github.io; object-src 'none'; manifest-src 'self';">
+    <meta http-equiv="Content-Security-Policy" content="connect-src 'self' ws://localhost:3000 https://catalogue.data.govt.nz https://api.addressfinder.io; style-src blob: https://api.addressfinder.io https://unpkg.com https://family.services.govt.nz 'unsafe-inline'; script-src 'self' https://api.addressfinder.io https://family.services.govt.nz https://unpkg.com 'unsafe-eval' 'unsafe-inline' 'unsafe-inline'; default-src 'none'; img-src 'self' data: https://a.tile.openstreetmap.org https://b.tile.openstreetmap.org https://c.tile.openstreetmap.org https://family.services.govt.nz https://unpkg.com; child-src 'none'; base-uri https://family.services.govt.nz; form-action https://family.services.govt.nz; object-src 'none'; manifest-src 'self';">
```

Comment:
Could we make the app allow *both* for now? . Then we can keep testing while waiting for tbe new domain.

Reply:
Should this be something determined by environment variables? There could be anything on localhost:3000 - that's probably not something that should be allowed in production!

Reply:
Seems reasonable. But i don't know how.
Should we generate it at build time?

Reply:
@lamorrison @JacOng17 Any ideas how to improve this and not have all the domains hard coded here?